packages/schemas/src/gen/utils.ts

```diff
@@ -6,6 +6,14 @@ import type { Field } from './types.js';
 export const normalizeWhitespaces = (string: string): string =>
   string.replaceAll(/\s+/g, ' ').trim();
 
+// eslint-disable-next-line unicorn/prevent-abbreviations -- JSDoc is a term
+export const stripLeadingJsDocComments = (string: string): string =>
+  string.replace(/^\s*\/\*\*[^*]+\*\//, '').trim();
```

Comment:
It doesn't seem to work with multi-line comments like this:
```
/**
* foo
*/
```

Reply:
How about `\/\*\*[\s\S]*?\*\/`

Reply:
yeah, was lazy

Reply:
actually this brings more issues than i expected. we need more effort to make the `.ts` output to be multi-line as well. i would like to keep as it is. the regexp itself can be optimized though